nsfw/nsfw.php

```diff
@@ -74,15 +74,15 @@ function nsfw_addon_settings(&$a, &$s)
 	}
 
 	$s .= '<span id="settings_nsfw_inflated" class="settings-block fakelink" style="display: block;" onclick="openClose(\'settings_nsfw_expanded\'); openClose(\'settings_nsfw_inflated\');">';
-	$s .= '<h3>' . L10n::t('Not Safe For Work (General Purpose Content Filter)') . '</h3>';
+	$s .= '<h3>' . L10n::t('Content Filter (NSFW tag and more)') . '</h3>';
```

Comment:
You should drop the "tag" after NSFW I think

Reply:
Will do.